### PR TITLE
Update EventContents_Specify modules 81X

### PR DESCRIPTION
**Pull request by jeongeun**
Update EventContents_Specify modules 81X (just a little)

### CODE REVIEW

**Comment by slava77** [comment]
(as a clarification)
The purpose of this PR is to keep explicit list of modules in the standard event content (used for jobs running in production).
With unscheduled execution, any stray module loaded in the jobs (e.g. by "import \*") and matching the keep pattern will run, which is not desired.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by mmusich** [comment]
@tocheng can you please have a look to the `ALCARECO` event content as done for the previous similar PR?

**Comment by slava77** [comment]
@cmsbuild please test
the comparisons never completed from the last test

File: Alignment/CommonAlignmentProducer/python/ALCARECOTkAlCosmicsInCollisions_Output_cff.py [hunk 1]
```diff
@@ -16,7 +16,9 @@
         'keep L1GlobalTriggerReadoutRecord_gtDigis_*_*',
         'keep *_TriggerResults_*_*',
         'keep DcsStatuss_scalersRawToDigi_*_*',
-        'keep Si*Cluster*_si*Clusters_*_*', # for cosmics keep original clusters
+        #'keep Si*Cluster*_si*Clusters_*_*', # for cosmics keep original clusters
+        'keep SiStripCluster_siStripClusters_*_*',
+        'keep SiPixelCluster_siPixelClusters_*_*',
```
Comment (slava77) [inline Alignment/CommonAlignmentProducer/python/ALCARECOTkAlCosmicsInCollisions_Output_cff.py]: this should be ``` SiStripCluster*_siStripClusters_*_* ```
the same for Pixel

File: CalibMuon/RPCCalibration/python/l1Muon_EventContent_cff.py [hunk 2]
```diff
@@ -2,9 +2,12 @@
```
Comment (slava77) [inline CalibMuon/RPCCalibration/python/l1Muon_EventContent_cff.py]: I didn't find this file included in standard workflows with RECO step.
We have other digi producers (simMuon\* ; hltMuon\*).

So, changes in this file should probably not be made.

File: CalibMuon/RPCCalibration/python/l1RPC_EventContent_cff.py [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 l1RPC_EventContent = cms.PSet(
     outputCommands = cms.untracked.vstring('drop *', 
-        'keep RPCDetIdRPCDigiMuonDigiCollection_*_*_*', 
+        #'keep RPCDetIdRPCDigiMuonDigiCollection_*_*_*', 
+        'keep RPCDetIdRPCDigiMuonDigiCollection_muonRPCDigis_*_*',
```
Comment (slava77) [inline CalibMuon/RPCCalibration/python/l1RPC_EventContent_cff.py]: I didn't find this file included in standard workflows with RECO step.
We have other digi producers (simMuon\* ; hltMuon\*).

So, changes in this file should probably not be made.

**Comment by slava77** [comment]
@mmusich 
I didn't find RpcCalHLT in any actively used ALCAs (at least not with git grep; das shows the last RpcCalHLT from 2010).
Is it still active in anything that includes RECO step?

The implementation is just an HLT filter. This means it can technically run on RAW inputs and non-physics streams, e.g. RPCMonitor.
If these use cases exist, the file ALCARECORpcCalHLT_Output_cff.py  should not be modified.

**Comment by slava77** [comment]
@jeongeun ping
Please follow up on the comments.
Thank you.

**Comment by slava77** [comment]
@jeongeun please clarify on your plans to update this PR.

**Comment by jeongeun** [comment]
Sorry for my late response.. I'll update as you commented and make new PR.

**Comment by slava77** [comment]
-1
superseded by #13948

**Comment by mmusich** [comment]
-1

**Pull request closed without merging**